manipulation/CMakeLists.txt

```diff
@@ -1,4 +1,4 @@
-cmake_minimum_required(VERSION 2.8.3)
+cmake_minimum_required(VERSION 2.8.12)
```

Comment:
@davetcoleman you added the `add_compile_options` line here, but not in `perception` and `robot_interaction`.
These two still do not build with c++11 afaics from this request.